topics/contributing/tutorials/create-new-tutorial-slides/slides.html

```diff
@@ -0,0 +1,332 @@
+---
+layout: tutorial_slides
+topic_name: contributing
+tutorial_name: create-new-tutorial-slides
+logo: "GTN"
+---
+
+# Slides
+
+- Introduce a subject
+- Quick overview
+- Not too much text
+- Images are Nice
+
+---
+
+
+
+## Content
+
+- Background information for the tutorial. E.g.:
+  - Motivation behind why X is useful
+  - Conepts needed to get most out of tutorial
+- Examples relevant to audience
+  - Avoid: "here is a feature to add link type tracks"
+  - Try for: "If you want to visualise SVs in cancer, ... this link type tracks are useful"
+  - Try for: Using examples from existing papers
+
+---
+
+
+## Formatting: Basic
+
+.pull-left[
+<span style="font-size: 35px;font-family: 'Yanone Kaffeesatz';font-weight: normal;text-align: center;">Slide Title</span>
+
+A list
+
+- Item 1
+- Item 2
+
+and numbered list
+
+1. Item
+2. Item
+
+Some text with a [link](https://example.com/) and an image:
+
+![Description of the image](/training-material/assets/images/GTN-60px.png)
+]
+
+.pull-right[
+```markdown
+### Slide Title
+
+A list
+
+- Item 1
+- Item 2
+
+and numbered list
+
+1. Item
+2. Item
+
+Some text with a [link](https://example.com/) and an image:
+
+![Description of the image](/training-material/assets/images/GTN-60px.png)
+---
```

Comment:
It may make sense to explain what are the `---` there. What do you think?

Reply:
Definitely, added a note and a presenter comment.